pkg/api/handlers.go

```diff
@@ -39,6 +43,35 @@ func newHandler(context *clusterd.Context, config *Config) *Handler {
 	}
 }
 
+func (h *Handler) Init() error {
```

Comment:
as far as i know, it's the unexported `init` function that has special meaning with the go runtime, but i have no problems renaming this function to reduce confusion